usr/share/rear/lib/layout-functions.sh

```diff
@@ -745,15 +745,15 @@ function get_part_device_name_format() {
     echo "$part_name"
 }
 #
-# apply-mappings function migrate disk device reference from an old system and
+# apply_layout_mappings function migrate disk device reference from an old system and
 # replace them with new one (from current system).
 # the relationship between OLD and NEW device is provided by $MAPPING_FILE
 # (usually disk_mappings file in $VAR_DIR).
-function apply-mappings() {
+function apply_layout_mappings() {
 
-    # apply-mappings need one argument (file which contains disk device to migrate).
+    # apply_layout_mappings need one argument (file which contains disk device to migrate).
     if [ -z $1 ] ; then
-        BugError "apply-mappings function called without argument (file_to_migrate)"
+        BugError "apply_layout_mappings function called without argument (file_to_migrate)"
```

Comment:
and then we can be more specific here and complain about *... called without argument or argument is not a non-empty file* :smile: